test/integration/targets/pip/tasks/pip.yml

```diff
@@ -122,6 +122,38 @@
     that:
       - "not url_installed.changed"
 
+# Test user and constraints installs
+- name: create a constraints file
+  copy: dest={{ output_dir }}/pipconstraints.txt
+    content="iso8601===0.1.11"
+
+- name: install iso8601 with constraints applied
+  pip:
+    name: iso8601
+    constraints: "{{ output_dir }}/pipconstraints.txt"
+    user: yes
+
+# TODO pip module does not report back version in the return info.
+# Once it does/can, change this to verify using req_installed.version instead
```

Comment:
Feel free to implement this :)
It shouldn't be too hard. We already return the version when given as input and the version is available in _is_present.

Reply:
Cool - I'll do that in a second PR.